_layouts/default-theme-wet-boew-fr.html

```diff
@@ -2,6 +2,11 @@
 <main property="mainContentOfPage" class="container">
 <h1 id="wb-cont" property="name">{{ page.title }}</h1>
 {% include content-buttons-fr.html %}
+{% if page.url contains 'accessibility' %}
```

Comment:
Instead of targeting the folder I think is preferable and more reusable to use an independant property like the PR I sent you.